examples/DHS-e2e/DHS/build.gradle

```diff
@@ -1,73 +1,73 @@
-    // on the Data Hub Framework library
-    compile 'com.marklogic:marklogic-data-hub:4.0.1'
-    compile 'com.marklogic:marklogic-xcc:9.0.7'
-    mlcp "com.marklogic:mlcp:9.0.7"
-    mlcp files("lib")
-}
-
-task importAllCustomers(type: com.marklogic.gradle.task.MlcpTask) {
-	doFirst {
-		classpath = configurations.mlcp
-		input_file_path = "/home/ec2-user/space/DHS/input/json/customers/"
-		input_file_type = "documents"
-		output_collections = "Customer,DHS"
-		output_permissions = "flowDeveloper,read,flowDeveloper,execute,flowDeveloper,insert,flowDeveloper,update,flowOperator,read,flowOperator,update,flowOperator,execute,flowOperator,insert" 
-		output_uri_replace = "/home/ec2-user/space/DHS/input/json,''" 
-		document_type = "json"
-		transform_module =  "/data-hub/4/transforms/mlcp-flow-transform.sjs"
-		transform_namespace = "http://marklogic.com/data-hub/mlcp-flow-transform"
-		transform_param = "entity-name=Customer,flow-name=custInput"
-		command = "IMPORT"
-		mode = "local"
-		host = "internal-mlaas-icalb-1owe6oc6du7uo-1452370097.us-east-1.elb.amazonaws.com"
-		port = 8006
-		username = "dhfoperator"
-		password = "M@rkl0gic"
-		restrict_hosts = true
-		}
-}
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This is a general purpose Gradle build.
+ * Learn how to create Gradle builds at https://guides.gradle.org/creating-new-gradle-builds/
+ */
+
+plugins {
+    id 'java'
+    id 'eclipse'
+    id 'idea'
+
+    // This plugin allows you to create different environments
+    // for your gradle deploy. Each environment is represented
+    // by a gradle-${env}.properties file
+    // See https://github.com/stevesaliman/gradle-properties-plugin
+    // specify the env on the command line with:
+    // gradle -PenvironmentName=x ...
+    id 'net.saliman.properties' version '1.4.6'
+
+    // This gradle plugin extends the ml-gradle plugin with
+    // commands that make the Data Hub Framework do its magic
+    id 'com.marklogic.ml-data-hub' version '4.0.3'
+}
+
+configurations {
+ mlcp {
+    resolutionStrategy {
+      force "xml-apis:xml-apis:1.4.01"
+    }
+  }
+}
+repositories {
+  jcenter()
+  mavenLocal()
+  // Needed for gradle plugins
+  maven { url "https://plugins.gradle.org/m2/" }
+  // Needed for mlcp dependencies
+  maven { url "https://developer.marklogic.com/maven2/" }
+  // Needed for hadoop dependencies for mlcp
+  maven { url "http://repository.cloudera.com/artifactory/cloudera-repos/" }
+}
+
+dependencies {
+    // this allows you to write custom java code that depends
+    // on the Data Hub Framework library
+    compile 'com.marklogic:marklogic-data-hub:4.0.3'
+    compile 'com.marklogic:marklogic-xcc:9.0.7'
+    mlcp "com.marklogic:mlcp:9.0.7"
+    mlcp files("lib")
+}
+
+task importAllCustomers(type: com.marklogic.gradle.task.MlcpTask) {
+	doFirst {
+		classpath = configurations.mlcp
+		input_file_path = "/Users/sbalasub/git/marklogic-data-hub/examples/DHS-e2e/DHS/input/json/customers/"

```

Comment:
How will you accommodate this use of your home directory?

Reply:
I have asked folks to edit the input file path in the README.